stdlib/@tests/test_cases/builtins/check_list.py

```diff
@@ -19,3 +19,8 @@ def asd(self) -> int:
 assert_type(combined, List[Union[Foo, Bar]])
 for item in combined:
     assert_type(item.asd(), int)
+
+l1: list[int] = [1]
+l2: list[object] = l1.copy()
```

Comment:
This should be an error, since `list[int]` shouldn't be able to be assigned to `list[object]`.